frontend/src/components/podcast/podcast.tsx

```diff
@@ -25,9 +25,8 @@ const Podcast: React.FC = () => {
           </div>
           <p className={styles.logoWrapper}>
             <img
-              src="#"
-              width="280"
-              height="280"
```

Comment:
This is inaccurate, but we have to declare the width and height in the attributes, as I understand it, so that there is no jump when the image is accidentally loaded.